test/adapter/z-stack/adapter.test.ts

```diff
@@ -3497,8 +3513,10 @@ describe('zstack-adapter', () => {
         basicMocks();
         await adapter.start();
         mockZnpRequest.mockClear();
-        bindStatusResponse = 1;
-        await expect(adapter.bind(301, '0x129', 1, 1, 4, 'endpoint', 9)).rejects.toThrow(`Failed to bind '1' from '4/9' to '0x129/1' (1)`);
+        bindStatusResponse = 0x8e;
+        await expect(adapter.bind(301, '0x129', 1, 1, 4, 'endpoint', 9)).rejects.toThrow(
+            `ZDO error: bind failed with status 'DEVICE_BINDING_TABLE_FULL' (142)`,
+        );
     });
```

Comment:
I don't see changes to test throwing for functions that now do? Coverage missed it?

Reply:
There were not tests for this yet, but given they all use `waitForAreqZdo` now I don't think it makes sense to add a bunch of tests for this?